api/compaction_manager.cc

```diff
@@ -116,9 +116,9 @@ void set_compaction_manager(http_context& ctx, routes& r) {
             table_names = map_keys(ctx.db.local().find_keyspace(ks_name).metadata().get()->cf_meta_data());
         }
         auto type = req->get_query_param("type");
-        co_await ctx.db.invoke_on_all([&ks_name, &table_names, type] (replica::database& db) {
+        co_await ctx.db.invoke_on_all([&] (replica::database& db) {
             auto& cm = db.get_compaction_manager();
```

Comment:
This looks fishy. So what if the inner lambda yields? There is co_await in the outer scope.

I think this is the lambda coroutine fiasco, and the patch just papers over it.